scripts/check_frontend_coverage.py

```diff
@@ -533,41 +587,48 @@ def check_coverage_changes():
         Exception: LCOV_FILE_PATH doesn't exist.
     """
     if not os.path.exists(LCOV_FILE_PATH):
-        raise Exception('Expected lcov file to be available at {}, but the'
-                        ' file does not exist.'.format(LCOV_FILE_PATH))
+        raise Exception(
+            'Expected lcov file to be available at {}, but the'
+            ' file does not exist.'.format(LCOV_FILE_PATH))
 
     stanzas = get_stanzas_from_lcov_file()
-    whitelist = list(FULLY_COVERED_FILENAMES)
+    blacklist_list = list(BLACKLIST)
```

Comment:
Actually, this is the wrong name. I suggested `remaining_blacklisted_files` before, since the idea is that this is a local variable that holds the files from the blacklist that remain to be looked at. The name blacklist_list is vague, and does not really make sense.

Reply:
Sorry about that! Done!